test/development/acceptance-app/hydration-error.test.ts

```diff
@@ -819,7 +819,7 @@ createNextDescribe(
 
       await cleanup()
     })
-    it.skip('should show correct hydration errror when server renders an extra Suspense node in content mode', async () => {
+    it('should show correct hydration errror when server renders an extra Suspense node in content mode', async () => {
```

Comment:
nit: typo on "error"
```suggestion
    it('should show correct hydration error when server renders an extra Suspense node in content mode', async () => {
```